docs/tutorials/handler.md

```diff
@@ -0,0 +1,389 @@
+Atomist uses Rug event handlers to listen to changes in your
+development systems and respond to them.  Each handlers is associated
+with specific events and when a matching event occurs, Atomist
+executes the event handler on that event.  A single event can match
+many event handlers.  In this example, we will create an event handler
+that sends a message to a GitHub repository's Slack channel when an
+issue in that repository is closed.
+
+## Create a Rug project
+
+The first thing you need is a project for your Rug event handler code.
+Although you can add Atomist Rug code to any existing project by
+simply adding a `.atomist` directory and a few supporting files, we'll
+use our preferred, faster, and less error prone method: using a
+generator.
+
+Ask the Atomist Bot to list the Rug generators by sending it the
+following message.
+
+```
+@atomist generators rug
+```
+
+You will see the following generators.
+
+<div class="ss-container">
+  <img src="../images/rug-generators.png" alt="Rug generators" class="ss-medium">
+</div>
+
+Select the "NewStarterRugProject" generator by clicking its "Create
+project" button and enter a name for the project when prompted by the
+Atomist Bot in the thread.
+
+<div class="ss-container">
+  <img src="../images/generate-starter-rug-project-thread.png" alt="Generate starter Rug project thread" class="ss-small">
+</div>
+
+After entering the name for the project, click on "Generate project"
+in the project generation summary:
+
+<div class="ss-container">
+  <img src="../images/generate-starter-rug-project-summary.png" alt="Generate starter Rug project summary" class="ss-small">
+</div>
+
+Atomist will then create the project your GitHub organization.
+
+<div class="ss-container">
+  <img src="../images/new-handlers-project-created.png" alt="New handlers project created message" class="ss-medium">
+</div>
+
+Atomist will also create a Slack channel in your team Slack for the
+repository.  The repository and channel will be connected, so when you
+join the new channel, you will see the initial commit made by the
+project generator.
+
+<div class="ss-container">
+  <img src="../images/handlers-slack-channel-created.png" alt="Handlers Slack channel initialized" class="ss-medium">
+</div>
+
+!!! caution ""
+    If you are working through these steps in
+    the [Atomist Community Slack][community], the new repository will
+    be created under your personal GitHub account and the Atomist Bot
+    will *not* create a Slack channel for the repository.
+
+[community]: https://join.atomist.com/
+
+Clone your new Rug project locally as you normally would.  The project
+will have contents like the following.
```

Comment:
I don't like having colons here because then you have a sentence without a closing period. But maybe that's just me.